loadgen/contract/agent-trade-amm.js

```diff
@@ -13,11 +13,11 @@ export default async function startAgent([key, home]) {
 
   console.error(`trade-amm: building tools`);
   // const runIssuer = await E(agoricNames).lookup('issuer', issuerPetnames.RUN);
-  const { runBrand, bldBrand, autoswap, runPurse, bldPurse } = await allValues({
+  const { runBrand, bldBrand, amm, runPurse, bldPurse } = await allValues({
     runBrand: E(agoricNames).lookup('brand', issuerPetnames.RUN),
     // bldBrand: E(agoricNames).lookup('brand', issuerPetnames.BLD),
     bldBrand: E(E(wallet).getIssuer(issuerPetnames.BLD)).getBrand(),
-    autoswap: E(agoricNames).lookup('instance', 'autoswap'),
+    autoswap: E(agoricNames).lookup('instance', 'amm'),
```

Comment:
Is there a way to make this backwards compatible in the loadgen? E.g. by getting both and using the one that resolves?

Reply:
I think this would work, but I haven't tested it.  Is it easy for you to verify that this would do you what you want?